ktor-http/ktor-http-cio/jvm/src/io/ktor/http/cio/Multipart.kt

```diff
@@ -479,8 +477,20 @@ private fun findBoundary(contentType: CharSequence): Int {
  * Parse multipart boundary encoded in [contentType] header value
  * @return a buffer containing CRLF, prefix '--' and boundary bytes
  */
-@KtorExperimentalAPI
+@Deprecated(
```

Comment:
replaceWith?

Reply:
There is no literal replacement at the moment so not sure if we need it to do that since the migration is not automatic anyway